src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25201.cs

```diff
@@ -0,0 +1,35 @@
+﻿#if ANDROID
```

Comment:
Could we run the test in all the platforms? In that way, we can just verify that the Padding property behavior is aligned.